src/pages/accounts/AdminAccountsPage.sagas.ts

```diff
@@ -1,28 +1,128 @@
-import { AxiosResponse } from "axios";
+import { AxiosRequestConfig, AxiosResponse } from "axios";
+import { PayloadAction } from "@reduxjs/toolkit";
 import { call, put, takeEvery } from "redux-saga/effects";
 import {
   requestAccounts,
   requestAccountsEnd,
   requestAccountsFail,
+  requestNewAccount,
+  requestNewAccountEnd,
+  requestNewAccountFail,
+  requestDeleteAccount,
+  requestDeleteAccountEnd,
+  requestDeleteAccountFail,
+  requestModifyAccount,
+  requestModifyAccountEnd,
+  requestModifyAccountFail,
+  setCurrentAccount,
+  setSelectedAccount,
+  unsetSelectedAccount,
 } from "./AdminAccountsPage.slice";
 import adminAPI from "../../app/common/apiClient";
-import { IAccount } from "./types";
+import {
+  INewAccount,
+  IAccount,
+  IAccountsGET,
+  IAccountPOST,
+  IAccountDELETE,
+} from "./types";
+import { IRequestMessage } from "../../shared/types";
 
 function* handleGetAccountsRequest(): any {
-  // request to be functional next PR
-  let accountsRequest = () => adminAPI.get<IAccount[]>("/v1/user/accounts");
+  let accountsRequest = () => adminAPI.get("/v1/user/getUsers");
+
+  try {
+    const { data }: AxiosResponse<IAccountsGET> = yield call(accountsRequest);
+
+    const { users, currentUser } = data;
+
+    const currentAccount = users.find((acc) => acc.username === currentUser);
+
+    yield put(requestAccountsEnd(users));
+    yield put(setCurrentAccount(currentAccount as IAccount));
+  } catch (e: any) {
+    const data: IRequestMessage = e.response.data;
+    const feedback: IRequestMessage = data.message ? data : e;
+    yield put(requestAccountsFail(feedback));
+  }
+}
+
+function* handleNewAccountRequest(action: PayloadAction<INewAccount>): any {
+  let accountsRequest = () => adminAPI.post("/v1/user/create", action.payload);
```

Comment:
nit: Wouldn't hurt to add "async () => ..."

You can test if it is currently running asynchronously by throwing a print statement in the function and a print statement after it is called I think. I'm just interested to see how it works.